### PR TITLE
MAT-7992: make improvement notation required for QI-Core

### PR DESCRIPTION
**Pull request by etan-sb**
## MADiE PR

Jira Ticket: [MAT-7992](https://jira.cms.gov/browse/MAT-7992)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by etan-sb** [comment]
Backend changes: https://github.com/MeasureAuthoringTool/measure-service/pull/766

**Review by mcmcphillips — DISMISSED** [review]

File: src/validations/MeasureGroupSchemaValidator.ts [hunk 1]
```diff
@@ -100,7 +100,10 @@ export const measureGroupSchemaValidator = (
   return Yup.object().shape({
     scoring: Yup.string()
       .oneOf(Object.values(GroupScoring))
-      .required("Group Scoring is required."),
+      .required("Group Scoring is required.ee"),
```
Comment (adongare) [inline src/validations/MeasureGroupSchemaValidator.ts]: looks like typo `.ee`

Comment (mcmcphillips) [reply src/validations/MeasureGroupSchemaValidator.ts]: I missed this 👍

Comment (etan-sb) [reply src/validations/MeasureGroupSchemaValidator.ts]: I pushed up a fix. Thanks!

**Review by adongare — APPROVED** [review]

**Pull request merged**